lib/config/migrations/custom/recreate-closed-migration.ts

```diff
@@ -0,0 +1,13 @@
+import is from '@sindresorhus/is';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class RecreateClosedMigration extends AbstractMigration {
+  override readonly deprecated = true;
+  override readonly propertyName = 'recreateClosed';
+
+  override run(value: unknown): void {
+    if (is.boolean(value)) {
+      this.setSafely('recreateWhen', value ? 'always' : 'auto');
```

Comment:
```suggestion
      this.setSafely('recreateWhen', value ? 'always' : 'never');
```
`recreateClosed=false` should be migrated to `recreateWhen=never` right? 
If the user uses this option, then they don't want the PR recreated ever.

Reply:
No. Since there was no such feature :), My bad